src/plugins/tonegen/indicator.h

```diff
@@ -0,0 +1,57 @@
+/*************************************************************************
+This file is part of tone-generator
+
+Copyright (C) 2010 Nokia Corporation.
+
+This library is free software; you can redistribute
+it and/or modify it under the terms of the GNU Lesser General Public
+License as published by the Free Software Foundation
+version 2.1 of the License.
+
+This library is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public
+License along with this library; if not, write to the Free Software
+Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301
+USA.
+*************************************************************************/
+
+#ifndef __TONEGEND_INDICATOR_H__
+#define __TONEGEND_INDICATOR_H__
+
+#ifndef _GNU_SOURCE
+#define _GNU_SOURCE
+#endif
+
+#include <stdint.h>
+
+#define PRESERVE_STREAM    0
```

Comment:
This could also be an enum (or a `bool` "kill stream"`, where "kill stream = false" means "preserve")